package.json

```diff
@@ -2,13 +2,26 @@
   "name": "ng-redux",
   "version": "3.4.1",
   "description": "Redux bindings for Angular.js",
-  "main": "./lib/index.js",
+  "browser": "dist/ng-redux.js",
+  "main": "lib/ng-redux.js",
+  "module": "es/ng-redux.js",
+  "typings": "./index.d.ts",
+  "files": [
+    "dist",
+    "lib",
+    "es",
+    "src",
+    "index.d.ts"
+  ],
   "scripts": {
-    "build": "npm run build:lib && npm run build:min && npm run build:max",
-    "build:lib": "rm -rf lib && babel src --out-dir lib",
-    "build:min": "webpack --optimize-minimize --define process.env.NODE_ENV=\"'production'\" lib/index.js dist/ng-redux.min.js",
-    "build:max": "webpack lib/index.js dist/ng-redux.js",
-    "test": "mocha --compilers js:babel-register --recursive"
+    "clean": "rimraf lib dist es",
+    "build:commonjs": "cross-env NODE_ENV=cjs rollup -c -o lib/ng-redux.js",
+    "build:es": "cross-env NODE_ENV=es rollup -c -o es/ng-redux.js",
+    "build:umd": "cross-env NODE_ENV=development rollup -c -o dist/ng-redux.js",
+    "build:umd:min": "cross-env NODE_ENV=production rollup -c -o dist/ng-redux.min.js",
+    "build": "$npm_execpath run build:commonjs && $npm_execpath run build:es && $npm_execpath run build:umd && $npm_execpath run build:umd:min",
```

Comment:
I want to mention that I'd rather we have this kind of compatibility for our yarn and NPM users than not but it seems like depending on that variable is not reliable :(